platform-sdk/sdk/settings.txt

```diff
@@ -31,16 +31,16 @@ event.creation.useTipsetAlgorithm,        true
                                           #############################
 
 event.enableEventStreaming,               false
-event.preconsensus.enableReplay,          false
-event.preconsensus.enableStorage,         false
+event.preconsensus.enableReplay,          true

```

Comment:
In https://github.com/hashgraph/hedera-services/pull/7399 , you explicitly deleted config values that matched the default.

Is there a reason you're leaving them here in this case?

Reply:
Yes. I leave config values that I think we may want to enable/disable during local testing. The ones I remove match the default AND are config values that we are unlikely to want to vary on any regular basis.